File: app/src/main/java/com/booboot/vndbandroid/model/vndb/CharacterVoiced.java
package com.booboot.vndbandroid.model.vndb;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class CharacterVoiced {
    private int id;
    private int aid;
    private int vid;
    private String note;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getAid() {
        return aid;
    }

    public void setAid(int aid) {
        this.aid = aid;
    }

    public int getVid() {
        return vid;
    }

    public void setVid(int vid) {
        this.vid = vid;
    }

    public String getNote() {
        return note;
    }

    public void setNote(String note) {
        this.note = note;
    }
}

File: app/src/main/java/com/booboot/vndbandroid/model/vndb/CharacterVoiced.kt
package com.booboot.vndbandroid.model.vndb

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class CharacterVoiced(
    var id: Int = 0,
    var aid: Int = 0,
    var vid: Int = 0,
    var note: String? = null
)